src/js/tech/html5.js

```diff
@@ -200,6 +200,26 @@ class Html5 extends Tech {
     });
   }
 
+  /**
+   * Attempt to force override of native audio tracks.
+   *
+   * @param {Boolean} override - If set to true native audio will be overridden,
+   * otherwise native video will potentially be used.
+   */
+  overrideNativeAudioTracks(override) {
+    this.forceNativeAudioOverride = override;
```

Comment:
should we just override `featuresNativeAudioTracks` and `featuresNativeVideoTracks`? While a lot of the docs language around it says it's around "support" it also means whether the tech will be using that feature or not.

Reply:
we may also need to do something like call `proxyNativeTracks_` after the fact if this is called with `false` so that that process kicks in, especially if we originally start with them disabled.